centreon/www/front_src/src/Dashboards/SingleInstancePage/Dashboard/AddEditWidget/AddEditWidgetModal.cypress.spec.tsx

```
@@ -55,25 +62,51 @@ const widgetsProperties = [
+  widgetResourceTableProperties,
+  widgetSingleMetricProperties,
+  widgetStatusGridProperties,
+  widgetStatusChartProperties
+];
 
-const platformVersion = {
+const getAvailableWidgets = (
+  defaultWidgetsProperties = widgetsProperties
+): Record<string, Version | null> =>
+  defaultWidgetsProperties.reduce(
+    (acc, { moduleName }) => ({ ...acc, [moduleName]: {} }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)